CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 6.2.0-dev
+
+* Stop generating a null-safety badge for elements
+  in a null-safe library. (#3295)
+* Use a sun icon for switching to the light theme. (#3309)
+* Standardize the search icon style to a new SVG. (#3302)
+* Remove CSS classes no longer used in dartdoc generated content. (#3302)
```

Comment:
This isn't generally user facing, but I suppose? there is potential for a downstream user to have been relying on them.

Reply:
👍